### PR TITLE
Feature/#8384 export certificates/#9064 info view

### PR DESCRIPTION
**Pull request by pascal-brause**
## Description
The info view when you tap "show printable version" in the action sheet of a health certificate.
When you tap "continue", the next screen will be the pdf of the certificate itself. This will be in another PR.

## Link to Jira
Mainstory: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8384
Subtask: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9064

## Screenshots
9064-one-light
9064-two-light
9064-three-light

9064-one-dark
9064-two-dark
9064-three-dark

### CODE REVIEW

**Review by InkenTownsend — APPROVED** [review]
UA approved

**Review by ArturFriesen — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/PdfGeneration/HealthCertificatePDFGenerationInfoViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+struct HealthCertificatePDFGenerationInfoViewModel {
+	
+	// MARK: - Internal
+	
+	let title: String = AppStrings.HealthCertificate.PrintPDF.Info.title
+	
+	var dynamicTableViewModel: DynamicTableViewModel {
+		DynamicTableViewModel([
+								
+			// Illustration with information text
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/PdfGeneration/HealthCertificatePDFGenerationInfoViewModel.swift:17]:
```suggestion
```

Not really necessary, no explanation needed

**Review by nickguendling — APPROVED** [review]

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/PdfGeneration/HealthCertificatePDFGenerationInfoViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+					)
+				]
+			)
+		])
+	}
+	
+	func generatePDFData(
+		completion: @escaping (PDFView) -> Void
+	) {
+		// will be completed correct with following PR
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/PdfGeneration/HealthCertificatePDFGenerationInfoViewModel.swift:47]: it's a todo or what? didn't get it really.

Comment (pascal-brause) [reply src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/PdfGeneration/HealthCertificatePDFGenerationInfoViewModel.swift:47]: Yeah, it is some kind of todo without marking it. In this function comes the in Arturs PR the pdf-creation-call.

**Pull request merged**